teams/templates/distribute_teams.html

```diff
@@ -0,0 +1,50 @@
+{% extends "base.html" %}
+{% load static %}
+{% load teams_tags %}
+{% block content %}
+
+<div class="h-100">
+    <article class="card shadow">
+        <div class="card-body">
+
+        {% if teams %}
+        {% for team_name, team_members in teams.items %}
+        <section class="card shadow">
+            <div class="card-header">{{team_name}}  <span style="float:right">Team Score: {{ team_members|calculate_team_level }}</span></div>
```

Comment:
Extra `space` before `<span>`. Also, try avoiding inline CSS where possible.

Reply:
Moved it to external css.